action/alarmWebAction.js

```diff
@@ -293,6 +293,9 @@ function main(params) {
                                 updatedParams.timezone = params.timezone;
                             }
                         }
+                        if(params.hasOwnProperty("strict")) {
```

Comment:
because `param.strict` 's type is `boolean`, so need to use hasOwnProperty to judge whether exist this filed.